core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/statistic/StatisticQueryProcessor.java

```diff
@@ -103,7 +103,7 @@ private String queryLimitCheck( String query ) {
     /**
      * Method to get all schemas, tables, and their columns in a database
      */
-    public ArrayList<ArrayList<String>> getSchemaTree() {
+    public List<List<String>> getSchemaTree() {
 
         ArrayList<ArrayList<String>> result = new ArrayList<>();
```

Comment:
My bad, I forgot to mention that also the type signatures in the method itself have to be adjusted.